clang/test/Driver/fp-model.c

```diff
@@ -163,23 +183,41 @@
 // CHECK-FEB-IGNORE: "-fno-rounding-math"
 // CHECK-FEB-IGNORE: "-ffp-exception-behavior=ignore"
 
-// RUN: %clang -### -nostdinc -Werror -ffast-math -ffp-model=fast -c %s 2>&1 \
+// RUN: %clang -### -nostdinc -Werror -ffast-math -ffp-model=aggressive -c %s 2>&1 \
+// RUN:   | FileCheck --check-prefix=CHECK-FASTMATH-FPM-AGGR %s
+// CHECK-FASTMATH-FPM-AGGR: "-cc1"
```

Comment:
perhaps pack some options on the same line to show that they are adjacent (no other option in between)

If this extra check isn't important, I expect at least `-SAME:`

Reply:
This is a direct copy of what this test was doing before for -ffp-model=fast. The "-SAME" check should work here. I'm not sure if the "-NOT" checks in the new -ffp-model=fast test below will work with "-SAME" but I can try that.